control/controlknobs/controlknobs.go

```diff
@@ -79,6 +83,8 @@ func (k *Knobs) UpdateFromNodeAttributes(selfNodeAttrs []string) {
 			forceBackgroundSTUN = true
 		case tailcfg.NodeAttrDisableDeltaUpdates:
 			disableDeltaUpdates = true
+		case tailcfg.NodeAttrPeerMTUEnable:
+			peerMTUEnable.Set(false)
```

Comment:
Just a quick thank-you to @noncombatant for noticing that this should be true and not false.